cmd/argo/commands/server.go

```diff
@@ -152,6 +158,7 @@ See %s`, help.ArgoServer),
 				RestConfig:               config,
 				AuthModes:                modes,
 				ManagedNamespace:         managedNamespace,
+				SSONamespace:             ssoNamespace,
```

Comment:
sorry - don't re-introduce this, instead add a func that determines it's value

Reply:
isn't it a better way to resolve `SSONamespace` like other parameters like `namespace` and `managedNamespace` in `server.go`
I am not introducing it as command line argument though.